securedrop/tests/test_unit_journalist.py

```diff
@@ -7,15 +7,17 @@
 import mock
 
 from flask_testing import TestCase
-from flask import url_for, escape, g
+from flask import url_for, escape
 
 import crypto_util
 import journalist
 import common
-from db import db_session, Source, Journalist, InvalidPasswordLength
+from db import (db_session, Source, Journalist, InvalidPasswordLength,
+                Submission)
 
 # Set environment variable so config.py uses a test environment
 os.environ['SECUREDROP_ENV'] = 'test'
+import config
```

Comment:
Ah, I see now. We don't actually access any of the attributes in config directly in this module. I have to refresh my memory on what happens if we import some other modules which import config before setting this environment variable. I think that could be bad, but need to confirm.